libraries/AP_HAL_ChibiOS/hwdef/common/stm32_util.c

```diff
@@ -468,3 +468,39 @@ uint32_t stack_free(void *stack_base)
     return ((uint32_t)p) - (uint32_t)stack_base;
 }
 #endif
+
+#ifdef RNG
+static uint32_t stm32_rand_generate(void)
+{
+    static uint32_t last_value = 0;
+    static uint32_t new_value = 0;
```

Comment:
doesn't need to be static for new_value

Reply:
ChibiOS changes moved to separate PR https://github.com/ArduPilot/ardupilot/pull/17855